packages/blade/src/components/BottomSheet/BottomSheet.web.tsx

```diff
@@ -33,28 +33,20 @@ import { useWindowSize } from '~src/hooks/useWindowSize';
 import { useIsomorphicLayoutEffect } from '~src/hooks/useIsomorphicLayoutEffect';
 import { useTheme } from '~components/BladeProvider';
 import { useId } from '~src/hooks/useId';
-import size from '~tokens/global/size';
+import { size } from '~tokens/global';
```

Comment:
Why are we changing this import like this?

Reply:
did this as a refactor because we already do a re-export from `global/index`